bundler/spec/quality_spec.rb

```diff
@@ -253,9 +253,62 @@ def check_for_specific_pronouns(filename)
     expect(lib_code).to eq(spec_code)
   end
 
+  it "documents all cli command options in their associated man pages" do
+    commands = normalize_commands_and_options(Bundler::CLI)
+    cli_and_man_pages_in_sync!(commands)
+
+    Bundler::CLI.subcommand_classes.each do |_, klass|
+      subcommands = normalize_commands_and_options(klass)
+
+      cli_and_man_pages_in_sync!(subcommands)
+    end
+  end
+
   private
 
   def each_line(filename, &block)
     File.readlines(filename, encoding: "UTF-8").each_with_index(&block)
   end
+
+  def normalize_commands_and_options(command_class)
+    commands = {}
+
+    command_class.commands.each do |command_name, command|
+      next if command.is_a?(Bundler::Thor::HiddenCommand)
+
+      key = command.ancestor_name || command_name
+      commands[key] ||= []
+      # Verify that all subcommands are documented in the main command's man page.
+      commands[key] << command_name unless command_class == Bundler::CLI
+
+      command.options.each do |name, option|
+        commands[key] << name.to_s.tr("_", "-") unless option.banner.to_s.include?("(deprecated)")
+      end
+    end
+
+    commands
+  end
+
+  def cli_and_man_pages_in_sync!(commands)
+    undocumented_options = ["help", "all"]
+
+    commands.each do |command_name, opts|
+      man_page_path = Dir.glob("#{__dir__}/../lib/bundler/man/bundle-#{command_name}*.ronn").first
```

Comment:
```suggestion
      man_page_path = man_tracked_files.find {|f| File.basename(f) == "bundle-#{command_name}.1.ronn" }
```

I think the above suggestion should fix the ruby-core job because it should be independent from the different directory setup when tests are run from ruby/ruby repo.